webpack.config.babel.js

```diff
@@ -76,14 +79,21 @@ const STAGING_CONFIG = {
 
 const CONFIG = PRODUCTION_API ? PROD_CONFIG : STAGING_CONFIG
 
+const formatDefineHash = defineHash =>
+  mapObject(
+    mapKeys(defineHash, key => `process.env.${key}`),
+    value => JSON.stringify(value)
+  )
+
 const basePlugins = [
   new webpack.NoEmitOnErrorsPlugin(),
-  new webpack.DefinePlugin({
-    'process.env.NODE_ENV': JSON.stringify(WEBPACK_ENV),
-    'process.env.ONFIDO_API_URL': JSON.stringify(CONFIG.ONFIDO_API_URL),
-    'process.env.ONFIDO_SDK_URL': JSON.stringify(CONFIG.ONFIDO_SDK_URL),
-    'process.env.SDK_VERSION': JSON.stringify(packageJson.version)
-  })
+  new webpack.DefinePlugin(formatDefineHash({
+    'NODE_ENV': WEBPACK_ENV,
+    'ONFIDO_API_URL': CONFIG.ONFIDO_API_URL,
+    'ONFIDO_SDK_URL': CONFIG.ONFIDO_SDK_URL,
+    'SDK_VERSION': packageJson.version,
+    'WOOPRA_DOMAIN': `${DEV_OR_STAGING? 'dev-':''}onfido-js-sdk.com`
```

Comment:
Minor: This spacing is ugly, prefer: `${DEV_OR_STAGING ? 'dev-' : ''}`

Otherwise looks good